src/App.test.tsx

```diff
@@ -5,7 +5,7 @@ import createReduxStore from "./redux/store/store";
 import initialState from "./redux/store/initialState";
 import { IApplicationState } from "./models//applicationState";
 import { mount } from "enzyme";
-import { HashRouter } from "react-router-dom";
+import { HashRouter, Router } from "react-router-dom";
```

Comment:
HashRouter not used anymore and we should be able to remove?

Reply:
yeah, it got added back with rebasing, will remove